README.md

```diff
@@ -14,18 +14,111 @@ All bugs, stories, and backlog for this project are managed through [Hyperledger
 in project IS (note that regular Indy tickets are in the INDY project instead...). Also, join
 us on [Hyperledger's Rocket.Chat](https://chat.hyperledger.org/) at #indy-sdk to discuss.
 
-## Building Indy SDK
+## How to install
+### Install for Ubuntu based distro (Ubuntu 16.04)
+It is recommended to install packages with APT:
+    
+    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 68DB5E88
+    sudo add-apt-repository "deb https://repo.sovrin.org/sdk/deb xenial {release channel}"
+    sudo apt-get update
+    sudo apt-get install -y libindy
+#### Release channels
```

Comment:
@vimmerru may be we can make "release channels" section common for all platforms?
+ do not duplicate info
- user may skip this section and start reading from platform specific

My personal vision is make this section common and make `{release channel}` as link to the section

Reply:
I believe release channels must be common section. And there can be:

{release channels} pointed pointed here may be one of these values: 
    
* `master` - for master release channel artifacts.
* `rc` - for rc release channel artifacts.
* `stable` - for stable release channel artifacts.

See <release-channels> section for details.